src/Builder/Creator/AbstractReflectionClass.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Builder\Creator;
+
+class AbstractReflectionClass
+{
+    /**
+     * @var string
+     */
+    protected const ARRAY_MERGE_FUNCTION = 'array_merge';
+
+    /**
+     * @var string
+     */
+    protected const RETURN_TYPE_STRING = 'string';
+
+    /**
+     * @var string
+     */
+    protected const RETURN_TYPE_ARRAY = 'array';
+
+    /**
+     * @var string
+     */
+    protected const RETURN_TYPE_BOOL = 'bool';
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    protected function isValueReturnArray($value): bool
+    {
+        return !is_bool($value)
+            && (is_array($value) || preg_match('/' . static::ARRAY_MERGE_FUNCTION . '/', $value));
+    }
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    protected function isValueReturnBool($value): bool
```

Comment:
why do we need a wrapper around simple function? It makes sense only when we give more context with a new name, but this is not a case

Reply:
but what we will do if expression will not like `true/false`? For example `$isValue && $isAnotherValue`. It's only for future improvments

Reply:
we can add it when we need it. RIght now this is just is_bool wrapper